src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationServiceTest.kt

```diff
@@ -60,18 +60,21 @@ class ActivitiesMigrationServiceTest {
   private val migrationHistoryService: MigrationHistoryService = mock()
   private val telemetryClient: TelemetryClient = mock()
   private val auditService: AuditService = mock()
-  val service = ActivitiesMigrationService(
+  val service = object : ActivitiesMigrationService(
     nomisApiService = nomisApiService,
-    queueService = queueService,
     activitiesMappingService = mappingService,
     activitiesApiService = activitiesApiService,
-    migrationHistoryService = migrationHistoryService,
-    telemetryClient = telemetryClient,
-    auditService = auditService,
     pageSize = 3L,
     completeCheckDelaySeconds = 10,
     completeCheckCount = 9,
-  )
+  ) {
+    init {
+      queueService = this@ActivitiesMigrationServiceTest.queueService
+      migrationHistoryService = this@ActivitiesMigrationServiceTest.migrationHistoryService
+      telemetryClient = this@ActivitiesMigrationServiceTest.telemetryClient
+      auditService = this@ActivitiesMigrationServiceTest.auditService
+    }
+  }
```

Comment:
This is how to set mocks that are autowired into the superclass instead of being passed into the constructor